{
	"id": 51284818,
	"number": 6196,
	"state": "closed",
	"title": "runtime: \"unexpected fault address 0x0\" during GC",
	"body": "by **mitchell.hashimoto**:\n\n\u003cpre\u003eGo version:\ngo version go1.1.2 linux/amd64\n\nWhile I haven't been able to reproduce this, a handful of users of my software have been\nseeing this regularly:\n\n\u003ca href=\"https://gist.github.com/mitchellh/6283509\"\u003ehttps://gist.github.com/mitchellh/6283509\u003c/a\u003e\n\nIt appears to be a SIGSEGV during hashmap access during GC. I'm sorry I can't provide\nmuch more information. If you need to point the stack traces back at my own code, here\nis the code in question (tagged so it won't change):\n\u003ca href=\"https://github.com/mitchellh/packer/tree/v0.3.3\"\u003ehttps://github.com/mitchellh/packer/tree/v0.3.3\u003c/a\u003e\u003c/pre\u003e",
	"user": {
		"login": "gopherbot",
		"id": 8566911,
		"type": "User",
		"site_admin": false
	},
	"labels": [
		{
			"name": "FrozenDueToAge"
		}
	],
	"comments": 15,
	"closed_at": "2015-05-12T21:11:10Z",
	"created_at": "2013-08-20T17:37:52Z",
	"updated_at": "2016-06-24T22:55:45Z",
	"milestone": {
		"id": 1055141,
		"number": 6,
		"title": "Unplanned"
	}
}
